Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
+
+  # lots of OS X support things were missing from this release
+  patch :DATA
```

Comment:
Can you please submit these changes upstream and link to the bug report or submission here? We do not accept patches that won't make it into the upstream sources. We'd definitely need a confirmation from upstream that this is the correct fix and that it will eventually make it into some future release. Thanks!